src/main/java/com/codurance/retropolis/controllers/BoardController.java

```diff
@@ -3,10 +3,17 @@
 
 import com.codurance.retropolis.config.GoogleTokenAuthenticator;
 import com.codurance.retropolis.entities.Board;
+import com.codurance.retropolis.config.GoogleTokenAuthenticator;
```

Comment:
Since you are using that `GoogleTokenAuthenticator`, can you please add the `Dev` profile to that class as well? Otherwise, the application will not start on dev mode. 
Here is a possible solution:

```
@Profile({Environment.PROD,Environment.DEV})
public class GoogleTokenAuthenticator implements HandlerInterceptor {
```